<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="predictButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>331</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fitButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fit Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> Air Quality Model Prediction</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>721</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Input Parameter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="pm10_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PM10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="so2_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SO2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>O3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="o3_input">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NO2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="no2_input">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>591</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: 500;
font-size: 20px;
</string>
   </property>
   <property name="text">
    <string> Air Quality :</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>predictButton</zorder>
  <zorder>fitButton</zorder>
  <zorder>label</zorder>
  <zorder>result_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
